### ISSUE TITLE
Two enters are required to run the previous command

### ISSUE THREAD

**Issue by blackteahamburger** [akinomyoga/ble.sh#382]
**ble version**: 0.4.0-devel4+a3b94bb
**Bash version**: 5.1.16(1)-release (x86_64-pc-linux-gnu)

After switching to the previous command, two enters are required to run it. For example:

```
$ [Up]
$ echo "$BLE_VERSION" [Enter] # No response
$ echo "$BLE_VERSION" [Enter][Enter]
0.4.0-devel4+a3b94bb
```
Or edit the command:

```
$ [Up]
$ echo "$BLE_VERSION **with $BASH_VERSION ($MACHTYPE)"** [Enter]
0.4.0-devel4+a3b94bb with 5.1.16(1)-release (x86_64-pc-linux-gnu)
```

**Comment by akinomyoga** [comment]
Do you have settings like the following?

```bash
bind '"\e[A": history-search-backward'
bind '"\e[B": history-search-forward'
```

Please use the following settings in `~/.blerc`. In particular, if you would like to accept the command by a single <kbd>Enter</kbd>, use the second one where `immediate-accept` is specified.

https://github.com/akinomyoga/ble.sh/blob/a3b94bb3b1f1e1d8e17ba2c02b3d4cf17bf10a8d/blerc.template#L949-L960

**Comment by akinomyoga** [comment]
Hmm, I actually intended to require <kbd>Enter</kbd> twice only when the history search is performed with a non-empty search string. In the case of the empty command line, I tried to make it behave the same as the readline bindable function `history-search-backward`.

I'll fix the behavior for the empty string case. However, please note that I'll keep the current behavior of requiring two <kbd>Enter</kbd> for the case of the non-empty string search.

**Comment by akinomyoga** [comment]
I forgot to tell you, but I have fixed it in commit 870ecef7d9ff8771e3dd6705994405e1ae92bef0 four days ago. Could you update ble.sh by running `ble-update` in a ble.sh session and check the behavior? Thank you!

**Comment by blackteahamburger** [comment]
> I forgot to tell you, but I fixed it in commit [870ecef](https://github.com/akinomyoga/ble.sh/commit/870ecef7d9ff8771e3dd6705994405e1ae92bef0) four days ago. Could you update ble.sh by running `ble-update` in a ble.sh session and check the behavior? Thank you!

Solved, thank you!

**Comment by akinomyoga** [comment]
Thank you for confirmation!
